saleor/plugins/webhook/plugin.py

```diff
@@ -358,6 +368,15 @@ def gift_card_deleted(self, gift_card: "GiftCard", previous_value: None) -> None
             WebhookEventAsyncType.GIFT_CARD_DELETED, gift_card
         )
 
+    def gift_card_metadata_updated(
```

Comment:
Each method added to any plugin that is used by PluginManager, should be also added to `BasePlugin`.